config.sub

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+# Canonicalize CHOST.  Like
+# https://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub
+# but accepts a much smaller set of machine specifications,
+# and is under the zlib license rather than the GPL.
+# See also https://bugs.gentoo.org/651908
```

Comment:
I don't know that we need to mention GPL here.

Reply:
Yeah, that could confuse license scanners.